### DISCUSSION TITLE
Upload datasets via UI

### DISCUSSION THREAD

**Discussion by assafweinberg** [langfuse/langfuse#1988] [Ideas]
### Describe the feature or potential improvement

I'd like to be able to upload and manage datasets via the UI, not just API.

**Comment by Ami777** [comment]
That one is veri important. Hope to see it soon!

**Comment by marcklingen** [comment]
Missed to update this thread, this was added in January: https://langfuse.com/changelog/2025-01-27-Dataset-Items-csv-upload

**Comment by talrejanikhil** [comment]
Is it possible to increase the file size limit to 50 MB from the current 10 MB?
